### PR TITLE
Allow to resume raps after speed up

### PR DESCRIPTION
**Pull request by brunobar79**
This PR adds the ability to call `executeRap` again  to resume the remaining tasks, which fixes the scenario of speeding up one of the transactions of a rap like : 
- unlock + swap
- unlock + swap + deposit

PoW: http://recordit.co/Vz3xQcrD5u

Fixes RNBW-1009

### CODE REVIEW

File: src/screens/SpeedUpAndCancelSheet.js [hunk 1]
```diff
@@ -99,6 +102,8 @@ const text = {
   [SPEED_UP]: `This will speed up your pending transaction by replacing it. There’s still a chance your original transaction will confirm first!`,
 };
 
+let existingWallet;
```
Comment (jinchung) [inline src/screens/SpeedUpAndCancelSheet.js:105]: Is there a reason this needs to be declared here?

**Comment by jinchung** [comment]
In the recording, it looks like we end up with 2 approvals in the txn list?

File: src/screens/SpeedUpAndCancelSheet.js (hunk 1 above)
Comment (brunobar79) [reply src/screens/SpeedUpAndCancelSheet.js:105]: Can't be declared in the sheet component scope cause the sheet gets unmounted after speeding up and the state is gone.

**Comment by brunobar79** [comment]
> In the recording, it looks like we end up with 2 approvals in the txn list?

There's only one. The other one was a failed test from a previous attempt

**Comment by brunobar79** [comment]
> In the recording, it looks like we end up with 2 approvals in the txn list?

The extra approval that shows up is because I revoked the allowance right before testing it outside rainbow and the tx shows up after the speed up confirmed when we reload the tx history from zerion.

If their API was working fine, that extra approval should have been there before I started the recording

**Review by skylarbarrera — APPROVED** [review]
looks good 👌

**Review by jinchung — APPROVED** [review]
🍨

**Pull request merged**